java/benchmarks/src/main/java/glide/benchmarks/clients/glide/GlideAsyncClient.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.benchmarks.clients.glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import glide.benchmarks.clients.AsyncClient;
+import glide.benchmarks.utils.ConnectionSettings;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/** A Glide client with async capabilities */
+public class GlideAsyncClient implements AsyncClient<String> {
+    private RedisClient redisClient;
+
+    @Override
+    public void connectToRedis(ConnectionSettings connectionSettings) {
+        if (connectionSettings.clusterMode) {
+            throw new RuntimeException("Use client GlideAsyncClusterClient");
+        }
+        RedisClientConfiguration config =
+                RedisClientConfiguration.builder()
+                        .address(
+                                NodeAddress.builder()
+                                        .host(connectionSettings.host)
+                                        .port(connectionSettings.port)
+                                        .build())
+                        .useTLS(connectionSettings.useSsl)
+                        .build();
+
+        try {
+            redisClient = RedisClient.CreateClient(config).get();
+        } catch (InterruptedException | ExecutionException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public CompletableFuture<String> asyncSet(String key, String value) {
+        return redisClient.customCommand(new String[] {"Set", key, value}).thenApply(r -> "Ok");
```

Comment:
why do you modify the returned value?

Reply:
Previous iterations of the client returned null. 
This is no longer true, and now the client returns "OK".  
Fixed.